src/Illuminate/Database/Connectors/PostgresConnector.php

```diff
@@ -81,34 +81,42 @@ protected function configureTimezone($connection, array $config)
     }
 
     /**
-     * Set the schema on the connection.
+     * Set the search_path on the connection.
      *
      * @param  \PDO  $connection
      * @param  array  $config
      * @return void
      */
-    protected function configureSchema($connection, $config)
+    protected function configureSearchPath($connection, $config)
     {
-        if (isset($config['schema'])) {
-            $schema = $this->formatSchema($config['schema']);
+        if (isset($config['searchpath']) || isset($config['schema'])) {
```

Comment:
Any reason why the config should be named `searchpath` when the postgres run-time parameter is called `search_path`?

E.g. the same class also deals with `application_name` which is named that way _in_ the configuration and in postgres.

Reply:
None at all! I had overlooked the fact that `application_name` contains an underscore, and concur that it should perhaps be `search_path`, for the sake of consistency. Nice catch!